docs/index.asciidoc

```diff
@@ -47,7 +47,11 @@ NOTE: Some multi-pipeline configurations such as logstash-to-logstash over http(
 
 Events that _fail_ ingest pipeline processing will be tagged with `_ingest_pipeline_failure`, and their `[@metadata][_ingest_pipeline_failure]` will be populated with details as a key/value map.
 
-IMPORTANT: This plugin requires minimum Java 17 and Logstash 8.7.0 versions.
+[id="plugins-{type}s-{plugin}-prerequisites"]
+===== Prerequisites
+- This plugin requires its major and minor versions to be the same with stack major and minor versions.
+- When upgrading the stack product(s) or plugin, make sure {es} and {kib} are upgraded to the target version before {ls} and <<plugins-{type}s-{plugin}>> plugin.
```

Comment:
Guidance needs _inversion_.

 - Given:
   - Agent deploys pipelines based on Kibana version
   - Upgrading Kibana before this plugin will cause Agent to deploy pipelines that this plugin cannot handle.
 - Therefore: this plugin must be upgraded _before_ Kibana.


```suggestion
- This plugin embeds portions of Elasticsearch Ingest Node whose version matches the `major`.`minor` version of the plugin.
- When upgrading the {stack} product(s), this plugin should be upgraded _before_ {kib} to ensure that {agent}-managed pipelines have access to the expected functionality.
```

We also do _not_ need to major/minor align with Logstash, so long as the JDK requirements are met. This plugin can be upgraded independently of Logstash.

Reply:
Good catch of `this plugin should be upgraded _before_ {kib}`, thank you!